.github/workflows/publish-latest.yml

```diff
@@ -134,3 +137,36 @@ jobs:
           value: 0
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: us-east-1
+
+  log-failure-metric:
+    # Send data point 1 to metric PublishLatestFailure in aws-amplify-ui+github-workflows@ ue-east-2, if it's a failure
```

Comment:
Mentioned in the other comment, but I'd prefer if we do not expose our account names. We can let secrets dictate which account for us.

Reply:
That make sense! I just removed them and changed the commit history. Let's delete the related comments so that it's not exposed in this PR